include/psen_scan_v2/udp_connection_state_machine.h

```diff
@@ -193,27 +224,31 @@ struct udp_connection_state_machine_ : public msm::front::state_machine_def<udp_
-    a_row < s::wait_for_monitoring_frame,  e::stop_request,                 s::wait_for_stop_reply,         &m::action_send_stop_request                                  >,
-      row < s::wait_for_stop_reply,        e::reply_received,               s::stopped,                     &m::action_notify_stop,             &m::guard_is_stop_reply   >
-    //  +--------------------------------+--------------------------------+-------------------------------+------------------------------------+-----------------------------+
+    //  +---------------------------------+--------------------------------+-------------------------------+-----------------------------------+-----------------------------+
+    a_row  < s::idle,                       e::start_request,                s::wait_for_start_reply,        &m::action_send_start_request                                 >,
+    a_row  < s::idle,                       e::stop_request,                 s::wait_for_stop_reply,         &m::action_send_stop_request                                  >,
+      row  < s::wait_for_start_reply,       e::reply_received,               s::wait_for_monitoring_frame,   &m::action_notify_start,            &m::guard_is_start_reply  >,
+    a_irow < s::wait_for_monitoring_frame,  e::monitoring_frame_received,                                    &m::action_handle_monitoring_frame                            >,
+    a_irow < s::wait_for_start_reply,       e::start_reply_timeout,                                          &m::action_send_start_request                                 >,
+    a_row  < s::wait_for_start_reply,       e::stop_request,                 s::wait_for_stop_reply,         &m::action_send_stop_request                                  >,
+    a_row  < s::wait_for_monitoring_frame,  e::stop_request,                 s::wait_for_stop_reply,         &m::action_send_stop_request                                  >,
+      row  < s::wait_for_stop_reply,        e::reply_received,               s::stopped,                     &m::action_notify_stop,             &m::guard_is_stop_reply   >
+    //  +---------------------------------+--------------------------------+-------------------------------+------------------------------------+-----------------------------+
   > {};
   // clang-format on
 
-  // LCOV_EXCL_START
-  // TODO: Activate coverage again when function is actually used
   // Replaces the default no-transition response.
   template <class FSM, class Event>
-  void no_transition(Event const&, FSM&, int)
+  void no_transition(Event const& event, FSM&, int state)
+  {
+    PSENSCAN_WARN("StateMachine", "No transition in state {} for event {}.", state, classNameShort(event));
+  }
+
+  template <class FSM>
+  void no_transition(e::monitoring_frame_received const& event, FSM&, int state)
   {
-    // TODO Implement handling
+    PSENSCAN_WARN("StateMachine", "Received monitoring frame despite not waiting for it");
```

Comment:
Could add in which state the monitoring frame was receive despite not waiting for it.